.github/workflows/neko.yml

```diff
@@ -14,4 +14,7 @@ jobs:
       - uses: Qiskit/qiskit-neko@main
         with:
           test_selection: backend
-          repo_install_command: pip install -e .
+          repo_install_command: |
+            pip install -r requirements-dev.txt
```

Comment:
This shouldn't be needed, neko's tests have their own dev requirements (which get installed by the action definition) so we should only need to install aer from source to be able to run the tests

Reply:
@mtreinish as shown in https://github.com/hhorii/qiskit-aer/actions/runs/3912462608/jobs/6687134955, installation of qiskit-aer fails because neko does not include `conan` in its `requirements-dev.txt`, I guess.

Reply:
@mtreinish I confirmed that `conan` command is installed but cmake does not recognize it as executable. In my local env, following logs are appeared in `pip install -v .` but not in [this CI](https://github.com/Qiskit/qiskit-aer/actions/runs/3928098538/jobs/6715355225).
```
    changing mode of /tmp/pip-build-env-hya3x7mj/overlay/bin/conan to 755
    changing mode of /tmp/pip-build-env-hya3x7mj/overlay/bin/conan_build_info to 755
    changing mode of /tmp/pip-build-env-hya3x7mj/overlay/bin/conan_server to 755
```
Usually, `pip` builds these commands in a temporary directory. I tried to change `TMPDIR` in aer installation but errors remained.

I would like install `conan` manually before qiskit-aer installation in neko. Could you give your thoughts?

Reply:
I think this is pointing to a wider issue in building from source. It looks like pip installing conan from pyproject.toml in that log but then cmake is not able to find it for some reason.

Reply:
The current pyproject.toml install `conan` correctly in my local docker environment. I guess that it does not work well only in github actions for `conan`.